app/controllers/conference_controller.rb

```diff
@@ -1,6 +1,13 @@
 class ConferenceController < ApplicationController
   before_filter :respond_to_options
   load_and_authorize_resource find_by: :short_title
+  skip_before_filter :verify_authenticity_token, if: :json_request?
```

Comment:
Ah there is an exception for ajax. Didn't know that. Please limit the skipping to the methods you really need it for. Not for the whole controller